app/knights/ammunition.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import annotations
+
+
+class Armour:
+    def __init__(self, part: str, protection: int) -> None:
+        self.part = part
+        self.protection = protection
+
+    @classmethod
+    def armours_from_dict(
+        cls, knight_name: str, knightsConfig: list[dict]
```

Comment:
don't use knightsConfig as name, because we don't use camel case in Python

Reply:
done